generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs

```diff
@@ -18,6 +18,7 @@
 -%>
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
+
```

Comment:
Those empty lines are intentional, those are separating library and application internal imports.

Reply:
yes intentional